CHANGELOG.md

```diff
@@ -1,5 +1,11 @@
 # Change log
 
+## master
+
+- Add `Cloner#partial_apply` method. ([@palkan][])
+
+- Add RSpec matchers `clone_association` / `clone_associations`. ([@palkan][])
```

Comment:
Can you add note about previous pr https://github.com/palkan/clowne/pull/17 too?)